projects/core/services/breakpoint.service.ts

```diff
@@ -0,0 +1,43 @@
+import {Inject, Injectable} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {fromEvent, merge, Observable} from 'rxjs';
+import {map, share, startWith} from 'rxjs/operators';
+
+import {TuiMedia} from '../interfaces';
```

Comment:
I believe you are supposed to use aliases like `@taiga-ui/core/interfaces` and `@taiga-ui/core/tokens`, perhaps that's why build is failing?